benchmarking/commons/launch_remote_local.py

```diff
@@ -110,42 +116,121 @@ def launch_remote(
         command line arguments
     :param extra_args: Extra arguments for command line parser, optional
     """
-    args, method_names, seeds = parse_args(methods, extra_args)
-    experiment_tag = args.experiment_tag
-    suffix = random_string(4)
-    benchmark = get_benchmark(args, benchmark_definitions)
-    master_random_seed = get_master_random_seed(args.random_seed)
+    if extra_args is None:
+        extra_args = []
+    else:
+        extra_args = extra_args.copy()
+
+    configuration = ConfigDict.from_argparse(
+        extra_args=extra_args + LOCAL_LOCAL_BENCHMARK_REQUIRED_PARAMETERS
+    )
+    launch_remote_banchmark(configuration, entry_point, methods, benchmark_definitions)
+
 
+def launch_remote_banchmark(
```

Comment:
"benchmark"

Reply:
I'd also change this name to `launch_remote_experiments`.

We run experiments, not benchmarks. The benchmarking is just the tuning problem, it is acted upon with a method and a backend in an experiment.